tests/installation/welcome.pm

```diff
@@ -193,6 +193,14 @@ sub run {
         }
         else {
             assert_and_click('before-select-product-' . $product);
+            if (!check_screen('select-product-' . $product)) {
+                if (check_var('VIDEOMODE', 'ssh-x') && is_ppc64le) {
```

Comment:
I don't think this can work. `check_screen` by default has a 0s timeout so this would only match in cases the screen is already showing the expected screen. Why do you need two nested if's and not just one statement

```suggestion
            if (!check_screen('select-product-' . $product) && check_var('VIDEOMODE', 'ssh-x') && is_ppc64le) {
```

?

This wouldn't solve the above problem though. In that case you could still add a reasonable timeout to the `check_screen` code

@coolo do you have a good idea how in this case we could avoid `check_screen` and `wait_still_screen` altogether but still work around the badly designed YaST installer that can at any time interfer with a popup that is preventing the previous click on the product selection to be properly caught?

Reply:
Well, as you already wrote: without the app behaving (showing its progress properly) I sense this is what a human would do too: wait until it stops acting strangely - and a little more.

But file a bug about it and let's hope for the best. That won't stop the QA code from working around it until we stop installing old products.